src/legacy/core_plugins/interpreter/public/interpreter.js

```diff
@@ -61,10 +61,11 @@ export const getInterpreter = async () => {
     _interpreterPromise = new Promise(resolve => _resolve = resolve);
     initialize();
   }
-  return await _interpreterPromise;
+  const { interpreter } = await _interpreterPromise;
+  return interpreter;
```

Comment:
@ppisljar I think this change is required in master as well- without unwrapping the ExpressionRenderer throws errors